client/src/screens/events/Detail.js

```diff
@@ -207,17 +218,18 @@ mapZoomMe = () => {
       // start a single fuzzy location fix aqusition
       navigator.geolocation.getCurrentPosition((position) => {
         this.processReturnedLocation(position, false);
+      }, (err) => {
+        console.log(err);
       });
 
       // start a accurate fix watcher
       this.geoWatch = navigator.geolocation.watchPosition(
         (position) => {
           this.processReturnedLocation(position, true);
+        }, (err) => {
+          console.log(err);
         },
```

Comment:
Alerts instead of console log?

Reply:
if I take that out I get a RedBox console.error popup. no idea what its doing it or how to stop it.